backends/graphics/openglsdl/openglsdl-graphics.h

```diff
@@ -97,6 +101,8 @@ class OpenGLSdlGraphicsManager : public OpenGL::OpenGLGraphicsManager, public Sd
 	uint _graphicsScale;
 	bool _gotResize;
 
+	bool _lockedScreen = false; // Variable to track lock state
```

Comment:
This variable name is not correct: what is locked is the windows size, nothing else.
The comment is useless.
Something like `_resizableWindow`  would fit better.

Reply:
Furthermore, you don't need a variable to track the lock state - SDL already holds that info.

You could do something like:

```
bool SdlGraphicsManager::isWindowResizable() const {
    return SDL_GetWindowFlags(_window->getSDLWindow()) & SDL_WINDOW_RESIZABLE;
}
```